### PR TITLE
android: add German translations

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/android/app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -4,73 +4,73 @@
...
-  <string name="deleted_description">deleted</string>
-  <string name="sending_files_not_yet_supported">sending files is not supported yet</string>
-  <string name="receiving_files_not_yet_supported">receiving files is not supported yet</string>
-  <string name="sender_you_pronoun">you</string>
-  <string name="unknown_message_format">unknown message format</string>
-  <string name="invalid_message_format">invalid message format</string>
+  <string name="deleted_description">Gelöscht</string>
+  <string name="sending_files_not_yet_supported">Das Senden von Dateien wird aktuell nicht unterstützt</string>
+  <string name="receiving_files_not_yet_supported">Der Empfang von Dateien wird aktuell nicht unterstützt</string>
+  <string name="sender_you_pronoun">Du</string>
```
Comment (epoberezkin) [inline apps/android/app/src/main/res/values-de/strings.xml:24]: Maybe, "Sie"?

Or is it common to use "Du" in the apps? In Russian translations we use the equivalent of "Sie"

File: apps/android/app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -4,73 +4,73 @@
...
-  <string name="description_you_shared_one_time_link_incognito">you shared one-time link incognito</string>
-  <string name="description_via_contact_address_link">via contact address link</string>
-  <string name="description_via_contact_address_link_incognito">incognito via contact address link</string>
-  <string name="description_via_one_time_link">via one-time link</string>
-  <string name="description_via_one_time_link_incognito">incognito via one-time link</string>
+  <string name="connection_local_display_name">Verbindung <xliff:g id="connection ID" example="1">%1$d</xliff:g></string>
+  <string name="display_name_connection_established">Verbindung hergestellt</string>
+  <string name="display_name_invited_to_connect">Für eine Verbindung eingeladen</string>
+  <string name="display_name_connecting">verbinde …</string>
+  <string name="description_you_shared_one_time_link">Du hast einen One-Time Link geteilt</string>
```
Comment (epoberezkin) [inline apps/android/app/src/main/res/values-de/strings.xml:33]: interesting that you decided not to translate "One-time link".

File: apps/android/app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -4,73 +4,73 @@
...
+  <string name="description_you_shared_one_time_link_incognito">Du hast einen Inkognito One-Time Link geteilt</string>
+  <string name="description_via_contact_address_link">Über einen Kontaktadressen Link</string>
+  <string name="description_via_contact_address_link_incognito">Über einen Inkognito-Kontaktadressen Link</string>
```
Comment (epoberezkin) [inline apps/android/app/src/main/res/values-de/strings.xml:36]: this is probably incorrect, in this case not the contact address link is incognito, but the user is. So "incognito" is a mode of connection here, not the property of the link

File: apps/android/app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -4,73 +4,73 @@
...
+  <string name="error_saving_smp_servers">Fehler beim Speichern des SMP Servers</string>
+  <string name="ensure_smp_server_address_are_correct_format_and_unique">Stelle sicher, dass die SMP Server Adressen das richtige Format haben, zeilenweise angeordnet und nicht kopiert sind.</string>
+  <string name="error_setting_network_config">Fehler bei der Aktualisierung der Netzwerk-Konfiguration</string>
 
   <!-- API Error Responses - SimpleXAPI.kt -->
-  <string name="contact_already_exists">Contact already exists</string>
-  <string name="you_are_already_connected_to_vName_via_this_link">You are already connected to <xliff:g id="contactName" example="Alice">%1$s!</xliff:g> via this link.</string>
+  <string name="contact_already_exists">Kontakt ist bereits vorhanden</string>
+  <string name="you_are_already_connected_to_vName_via_this_link">Du bist bereits über diesen Link mit <xliff:g id="contactName" example="Alice">%1$s!</xliff:g> verbunden.</string>
   <string name="invalid_connection_link">Invalid connection link</string>
```
Comment (epoberezkin) [inline apps/android/app/src/main/res/values-de/strings.xml:48]: skipped

File: apps/android/app/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -4,73 +4,73 @@
...
+  <string name="it_can_disabled_via_settings_notifications_still_shown"><b>Kann über die Einstellungen deaktiviert werden</b> – Solange die App abläuft werden Benachrichtigungen weiterhin angezeigt.</string>
+  <string name="turn_off_battery_optimization">Um diese Funktion zu nutzen, ist es nötig die Einstellung <b>Akkuoptimierung</b> für <xliff:g id="appName">SimpleX</xliff:g> im nächsten Dialog zu <b>deaktivieren</b>. Ansonsten werden die Benachrichtigungen deaktiviert.</string>
+  <string name="turning_off_service_and_periodic">Die Akkuoptimierung ist aktiv, der Hintergrunddienst und die regelmäßige Nachfrage nach neuen Nachrichten ist abgeschaltet. Du kannst diese Funktion wieder in den Einstellungen aktivieren.</string>
+  <string name="periodic_notifications">Regelmäßige Benachrichtigungen</string>
+  <string name="periodic_notifications_disabled">Regelmäßige Benachrichtigungen sind deaktiviert!</string>
+  <string name="periodic_notifications_desc">Die App holt regelmäßig neue Benachrichtigungen ab — dies benötigt ein paar Prozent Akkuleistung am Tag. Die App nutzt keine Push-Benachrichtigungen — es werden keine Daten von Deinem Gerät an Server gesendet.</string>
```
Comment (epoberezkin) [inline apps/android/app/src/main/res/values-de/strings.xml:69]: I think it's better to say the app fetches "messages", not "notifications"

**Review by Nisutec — CHANGES_REQUESTED** [review]
Hello,
I have checked the translation and have found some problems / wrong translations.

In my opinion the following translations are better translated into German:
``` xml
<!-- PendingContactConnection - ChatModel.kt -->
<string name="description_you_shared_one_time_link">Sie haben einen Einmal-Link geteilt</string>
<string name="description_you_shared_one_time_link_incognito">Sie haben Inkognito einen Einmal-Link geteilt</string>
<string name="description_via_one_time_link">Über einen Einmal-Link</string>
<string name="description_via_one_time_link_incognito">Inkognito über einen Einmal-Link</string>

<!-- API Error Responses - SimpleXAPI.kt -->
<string name="connection_timeout">Verbindungszeitüberschreitung</string>
<string name="connection_error">Verbindungsfehler</string>
<string name="network_error_desc">Bitte überprüfen Sie Ihre Netzwerkverbindung und versuchen Sie es erneut.</string>
<string name="error_sending_message">Fehler beim Senden der Nachricht</string>
<string name="error_adding_members">Fehler beim Hinzufügen von Mitgliedern</string>
<string name="error_joining_group">Fehler beim Beitritt der Gruppe</string>
<string name="cannot_receive_file">Datei kann nicht empfangen werden</string>
<string name="sender_cancelled_file_transfer">Der Absender hat die Dateiübertragung abgebrochen.</string>
<string name="error_receiving_file">Fehler beim Empfangen der Datei</string>
<string name="error_creating_address">Fehler beim Erstellen der Adresse</string>
<string name="error_accepting_contact_request">Fehler beim Akzeptieren der Kontaktanfrage</string>
<string name="sender_may_have_deleted_the_connection_request">Der Absender hat möglicherweise die Verbindungsanfrage gelöscht.</string>

<!-- ChatListView.kt -->
<string name="tap_to_start_new_chat">Tippen Sie um einen neuen Chat zu starten</string>
<string name="chat_with_developers">Chatten Sie mit den Entwicklern</string>
<string name="you_have_no_chats">Sie haben keine Chats</string>

<!-- NewChatSheet -->
<string name="share_one_time_link">Erstellen Sie einen einmaligen Einladungslink</string>
<string name="connect_via_link_or_qr">Per Link / QR-Code verbinden</string>

 <!-- Actions - ChatListNavLinkView.kt -->
<string name="mute_chat">Stummschalten</string>
<string name="unmute_chat">Stummschaltung aufheben</string>

<!-- PasteToConnect.kt -->
<string name="connect_button">Verbinden</string>
<string name="paste_button">Einfügen</string>

<!-- CreateLinkView.kt -->
<string name="one_time_link">Einmaliger Einladungslink</string>
<string name="your_contact_address">Ihre Kontaktadresse</string>

<!-- Default themes -->
<string name="theme_light">Hell</string>
```

And I have found one problem at line 53 behind </string> close tag. There is text behind the close tag.

**Comment by mlanp** [comment]
@Nisutec:
Thank you for your valuable support. I added your translations and corrections. The push will be done this night.
You have to know, that the German translations are an ongoing and dynamic process at the moment. Therefore some of the strings are not or wrong translated yet. The plan is to finish this process with the release of v4.

**Comment by Nisutec** [comment]
If you need again help on translating or reviewing feel free to mention me.

**Pull request merged**